themes/default/layouts/robots.txt

```diff
@@ -5,3 +5,6 @@ Disallow: *
 
 # Disallow certain pages that are preceived as soft-404s.
 Disallow: /docs/search/$
+
+Sitemap: https://www.pulumi.com/sitemap.xml
+Host: https://www.pulumi.com
```

Comment:
Looking at https://developers.google.com/search/docs/advanced/robots/robots_txt#syntax, I don't see `Host` listed as a property Google supports, and I don't see it listed in the spec at https://datatracker.ietf.org/doc/html/draft-rep-wg-topic-00 (assuming I'm not looking in an outdated place). @Brant-Soler is there guidance that explains this prop in more detail?

Reply:
Looks like based on [this recommendation](https://yoast.com/ultimate-guide-robots-txt/#host-directive), we should probably omit the protocol bit:

```suggestion
Host: www.pulumi.com
```